### PR TITLE
Add open graph tags to projects [mozilla/webmaker-core#453]

### PR DESCRIPTION
**Pull request by errietta**

### CODE REVIEW

**Comment by thisandagain** [comment]
What about using something like react-helmet? https://github.com/nfl/react-helmet

**Comment by errietta** [comment]
@thisandagain  thing is then we would need to write the server using react instead. This has to be done srerver side as facebook's spider doesn't  do js

File: server.js [hunk 1]
```diff
@@ -34,12 +36,52 @@ server.register({
...
+        method: 'GET',
+        uri: config.API_URI +
+        '/users/' + request.query.user +
+        '/projects/' + request.query.project,
+        json: {}
+      };
+
+      nets(options, function (err, res, body) {
+        if (err || res.statusCode !== 200) {
+          return;
```
Comment (k88hudson) [inline server.js:67]: You need to reply with something here. As long as the error is handled on the client, just return `reply.file('index.html')`

**Comment by errietta** [comment]
@k88hudson re-r?

**Comment by k88hudson** [comment]
@errietta AWESOME!!!! :dancer: 

image

R+. I'm gonna file a follow-up ticket for re-writing the URL to be prettier, as well as a ticket in core to switch over the share link

**Comment by xmatthewx** [comment]
@secretrobotron will be _very_ excited to see this

**Pull request merged**